website/addons/wiki/templates/edit.mako

```diff
@@ -111,7 +111,7 @@
               </div>
           </div>
 
-          % if user['can_edit']:
+          % if has_permission_or_publicly_editable:
```

Comment:
This variable belongs with with the other permissions variables, nested under the `user` variable. In other words, the data from the view should include something like user['can_edit_wiki'].